### PR TITLE
Convert to use Pipenv

### PR DESCRIPTION
**Pull request by bseb**
### Background
Convert to using Pipenv to align with Panther-Analysis

### Changes
- Update Pipfile
- Update Makefile
- Update ci config
-  ~~Remove requirements.txt~~
requirements.txt is still used by parts the CI config

- Added a helper script to update version numbers quickly

### Testing
CI/CD builds, manual testing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,32 +1,30 @@
 packages = panther_analysis_tool
 
-ci: lint unit integration
+ci:
+	pipenv run $(MAKE) lint unit integration
```
Comment (lindsey-w) [inline Makefile:4]: does this end up running `pipenv run make lint` -> `pipenv run pipenv run mypy...`? Curious if we should remove that nesting

Comment (bseb) [reply Makefile]: Good catch. That Pipenv run doesn't need to be there. Removed.

Comment (bseb) [reply Makefile:4]: Interestingly CI fails without this, I'm not sure why yet

Comment (bseb) [reply Makefile]: Resolved. Found a pip install that was not running in the pipenv environment. Corrected and now works as expected

**Review by lindsey-w — APPROVED** [review]

**Pull request merged**